### PR TITLE
Clean up (part 1)

### PR DESCRIPTION
**Pull request by felixvd**
This only changes type hints and moves one function up. Should not touch any runtime code.

This is preparation for generating the client.

### CODE REVIEW

File: mujinvisioncontrollerclient/visioncontrollerclient.py [hunk 1]
```diff
@@ -20,24 +22,24 @@ class VisionControllerClient(object):
...
+    _configurationsocket = None  # type: Optional[zmqclient.ZmqClient]
     
     _callerid = None # the callerid to send to vision
     _checkpreemptfn = None # called periodically when in a loop
     
     _subscriber = None # an instance of ZmqSubscriber, used for subscribing to the state
     
     def __init__(self, hostname='127.0.0.1', commandport=7004, ctx=None, checkpreemptfn=None, reconnectionTimeout=40, callerid=None):
-        # type: (str, int, typing.Optional[zmq.Context]) -> None
+        # type: (str, int, Optional[zmq.Context], Optional[Callable], int, Optional[str]) -> None
```
Comment (florisatmujin) [inline mujinvisioncontrollerclient/visioncontrollerclient.py]: Below in the docstring:
```
reconnectionTimeout (float, optional): Sets the "timeout" parameter of the ZmqSocketPool instance
```

is it float or int?

File: mujinvisioncontrollerclient/visioncontrollerclient.py [hunk 2]
```diff
@@ -427,7 +443,7 @@ def WaitForGetLatestDetectionResultImages(self, timeout=2.0):
         return self._WaitForResponse(recvjson=False, timeout=timeout)
     
     def GetDetectionHistory(self, timestamp, timeout=2.0):
-        # type: (float, float) -> typing.Any
+        # type: (float, float) -> Any
```
Comment (florisatmujin) [inline mujinvisioncontrollerclient/visioncontrollerclient.py]: Docstring suggest that timestamp should be `timestamp (int): unix timestamp in milliseconds`

File: mujinvisioncontrollerclient/visioncontrollerclient.py [hunk 3]
```diff
@@ -452,17 +468,8 @@ def GetVisionStatistics(self, taskId=None, cycleIndex=None, taskType=None, timeo
...
-        if not fireandforget:
-            return self._ProcessResponse(response, command=configuration, recvjson=recvjson)
-        return response
-    
     def Ping(self, timeout=2.0):
-        # type: (float) -> typing.Dict
+        # type: (float) -> Optional[Dict]
         return self._SendConfiguration({"command": "Ping"}, timeout=timeout)
     
     def SetLogLevel(self, componentLevels, timeout=2.0):
```
Comment (florisatmujin) [inline mujinvisioncontrollerclient/visioncontrollerclient.py:475]: no type definition?

File: mujinvisioncontrollerclient/visioncontrollerclient.py (hunk 1 above)
Comment (felixvd) [reply mujinvisioncontrollerclient/visioncontrollerclient.py]: float, fixed

File: mujinvisioncontrollerclient/visioncontrollerclient.py (hunk 2 above)
Comment (felixvd) [reply mujinvisioncontrollerclient/visioncontrollerclient.py]: That's correct afaik, the type hint was wrong

File: mujinvisioncontrollerclient/visioncontrollerclient.py (hunk 3 above)
Comment (felixvd) [reply mujinvisioncontrollerclient/visioncontrollerclient.py:475]: Not for now. When we get to it we'll do it right.

Comment (felixvd) [reply mujinvisioncontrollerclient/visioncontrollerclient.py:475]: I left the existing type hints in here, but the rest will come when they're generated

**Pull request merged**